src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/ISelectionService.cs

```diff
@@ -13,7 +13,7 @@ public interface ISelectionService
         /// <summary>
         /// Gets the object that is currently the primary selection.
         /// </summary>
-        object PrimarySelection { get; }
+        object? PrimarySelection { get; }
```

Comment:
https://github.com/dotnet/runtime/blob/339a204e35e7c6ed66c0a4ee86543dd282fa46b4/src/libraries/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs#L1946 needs to be updated as well. Thanks.

Reply:
done.